v3/src/components/case-table/use-columns.tsx

```diff
@@ -32,10 +31,11 @@ export function renderValue(str = "", num = NaN, attr?: IAttribute) {
   const { type, userType } = attr || {}
 
   // colors
-  if ((type === "color" && parseColor(str).space) || (!userType && parseColorStrict(str).space)) {
+  const color = type === "color" || !userType ? parseColor(str, { colorNames: type === "color" }) : ""
+  if (color) {
     return (
       <div className="cell-color-swatch" >
-        <div className="cell-color-swatch-center" style={{ background: str }} />
+        <div className="cell-color-swatch-center" style={{ background: color }} />
```

Comment:
I think this will make `background: true` if `attr.type === "color"`, which seems wrong.